### DISCUSSION TITLE
关于量算组件示例中 distance 与 polyline 测量结果的问题

### DISCUSSION THREAD

**Discussion by wanfzz** [zouyaoji/vue-cesium#579] [Q&A]
![image](https://github.com/zouyaoji/vue-cesium/assets/45428227/3d4f81d1-46d9-43f7-b75a-31ac595447f1)
折现距离和距离得到的值基本一致，我理解的折现距离应该是把起伏部分计算在内？想问一下这部分是bug还是需要自己手动计算更新label的值

**Comment by zouyaoji** [comment]
对，没有考虑哈，直接用Cartesian3.distance计算的。

不知道咋计算啊，可能要去用很多点去采样出高度，然后依次算。

**Reply by wanfzz** [reply]
ok
